production-vms/go.mod

```diff
@@ -0,0 +1,32 @@
+module production-vms
```

Comment:
sorry you know me I'm a sticker on naming...

`github.com/Azure/agentbaker/production-vms`

also why `production-vms` in the first place?

Reply:
naming is definitely open to change. Its VMs of our images in production, so that was the thought behind the name lol

Reply:
I think it should be named with their intent.   `vulnerability-reporting-vms` ?